src/Srclib/Converter.hs

```diff
@@ -35,7 +35,7 @@ toSourceUnit ProjectResult{..} =
           }
     }
   where
-    renderedPath = Text.pack (toFilePath projectResultPath) <> "||" <> projectResultType
+    renderedPath = Text.pack (toFilePath projectResultPath)
```

Comment:
Does this invalidate or fix the `TODO` on line 28:
```haskell
sourceUnitManifest = renderedPath, -- TODO: use a real value here instead of renderedPath?
```

Reply:
It does 👍